Update documentation to 0.902 (#11)
Neogoma/stardust-SDK-doc: update.md

# Update log

## 0.902
* **SDK**
    * feature: URL's can now be used for displaying pictures
* **Dashboard**
    * feature: allowing users to choose between image URL or GLB upload
* **API (premium feature)**
    * feature: Create a picture via API

## 0.901
* **SDK**
    * fix: correct the bug on different object creation

## 0.9
* **SDK**
    * feature: Adding the scripts for offline mapping
* **API (premium feature)**
    * feature: Download point cloud function 
    * feature: Get map batch data

## 0.85
* **SDK**

>     * feature: Sending necessary datas for analytics

Reviewer: ```suggestion
    * feature: Sending necessary data for analytics
```

* **Dashboard**
    * feature: Analytics editor for paying users

## 0.81
* **SDK**
    * fix: object rotation on relocation
    * fix: multiple object creations on the map download

## 0.8
* **SDK**
    * feature: Using GLB files
* **Dashboard**
    * update: Updating the object upload page to manage GLB files
    * update: Editor upgrade

>     * feature: Allow users download their failed relocations (premium feature)

Reviewer: ```suggestion
    * feature: Allow users to download their failed relocation pictures (premium feature)
```

    
## 0.772
* **SDK**
    * update: Updating to Hobodream 1.6

## 0.771
* **SDK**
    * update: Updating the Object management method names to show the MVC structure
    * fix: Correcting the update coordinates after relocation
    * update: Adding methods in object controller to manage the metadata

## 0.77
* **SDK**
    * update: Adding metadata field in the persistent object's data.

## 0.76
* **SDK**

>     * update: Adding the property RelocationReliable on the relocation results to feedback user on the relocation results

Reviewer: ```suggestion
    * update: Adding the property `RelocationReliable` on the relocation results to feedback user on the relocation results
```

## 0.75
* **SDK**
    * update: Increasing relocation accuracy

## 0.7
* **SDK**
    * feature: Set up call back for maximum number of requests
    * feature: Allow relocation without downloading the map    

## 0.65
* **SDK**
    * update: Reducing capture memory leaks

>     * update: Updating the file structure to allow customs data provider (for non AR foundation datas)

Reviewer: ```suggestion
    * update: Updating the file structure to allow custom data provider (for non AR foundation data)
```
